src/api/app/models/event/relationship_create.rb

```diff
@@ -4,6 +4,12 @@ class RelationshipCreate < Relationship
     self.description = 'Relationship was created'
 
     receiver_roles :any_role
+
+    def originator
+      return User.find_by(login: payload['who']) if payload['who']
+
+      nil
```

Comment:
I believe this `nil` is redundant, as if the condition above is not met it will return nil automatically. I'm not 100% sure if this is more or less readable... Thoughts?

Reply:
You are right, it will return `nil` anyway whenever `if payload['who']` evaluates to `false`.